composer.json

```diff
@@ -3,9 +3,10 @@
   "description": "A drop-in replacement for mysqldump that optionally sanitizes DB fields for better GDPR conformity.",
   "require": {
     "symfony/console": "^3.0",
-    "ifsnop/mysqldump-php": "^2.4",
+    "ifsnop/mysqldump-php": "dev-master",
```

Comment:
shouldn't we lock it down to a version that we know it works and it doesn't break changes? thinking of sussy, csswring... we might target the whole 2.X as that _shouldn't_ break but if the library has a breaking change (ie: 3.1) it might affect ours.

Reply:
@fjgarlin right now the changes we need aren't in a release (the hooks haven't been incorporated in a tagged version yet). That's why we're using dev-master.
We could make it attach to a commit?

Reply:
I guess either attach to commit or leaving like this then. We'll keep track of the other repo and should anything break we'd know where to look first :-)

Reply:
100% - I'm monitoring mysqldump-php closely, as soon as the new release is out, I'll update it.
I'll open an issue now to remind us.